client/app/lib/dateTimeDefaults.js

```diff
@@ -1,19 +1,29 @@
-import moment from 'moment';
+import moment from 'lib/moment';
+
+const timeZone = document.head.querySelector("[name='server-context']")
+                     .getAttribute('data-time-zone');
 
 export const shortDateFormat = {
   year: 'numeric',
   month: 'numeric',
   day: 'numeric',
+  timeZone,
```

Comment:
Yes, this is used where react-intl is bypassed and intl.formatDate or intl.formatTime is called directly

[Example](https://github.com/Coursemology/coursemology2/blob/b1b34a5125e4d1accd43fc9e1d90b99c33852149/client/app/bundles/course/lesson_plan/components/LessonPlanItem.jsx#L172)